omrversionstrings.CMakeTemplate.h

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2015
```

Comment:
This is a written file that is used to generate `omrversionstrings.h`. Will update the copyright.